resources/default.rb

```diff
@@ -49,6 +49,9 @@
     compile_time false
   end
 
+  chef_gem 'mixlib-shellout' do
+  end
```

Comment:
I think the gem is bundled by default with chef

Reply:
So should I remove it, or we can consider it being safer to have this?